integration-tests/src/test/resources/features/action-sendtobridge.feature

```diff
@@ -0,0 +1,60 @@
+Feature: SendToBridge Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "bridge1"
+    And the Bridge "bridge1" is existing with status "ready" within 2 minutes
```

Comment:
Please raise timeout for bridges creation to 4 minutes as done in other scenarios (due to changes in worker processing).